tests/study/test_study_models.py

```diff
@@ -4,6 +4,7 @@
 from dataservice.api.participant.models import Participant
 from dataservice.api.study.models import Study
 from tests.utils import FlaskTestCase
+from unittest.mock import patch, Mock
 
 
 class ModelTest(FlaskTestCase):
```

Comment:
Nothing happens. The bucket will remain so as to avoid any unintentional data loss. May be worth adding logic to the bucket service to remove buckets only if they are empty and don't exist in the dataservice.